tensorpack/train/trainers.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import tensorflow as tf
 
+from horovod.tensorflow.compression import Compression
```

Comment:
Cannot import here because horovod is not a required dependency of tensorpack.

Reply:
Will put a try catch block.

Reply:
Please put the import where horovod is currently imported

Reply:
Move into the HorovodTrainer